package.json

```diff
@@ -36,7 +36,7 @@
     "playwright:run": "yarn run playwright test playwright/e2e",
     "playwright:debug": "yarn run playwright test playwright/e2e --debug",
     "playwright:demo": "SLOW_MO=750 TIMEOUT=600000 yarn run playwright test playwright/e2e --headed",
-    "test:e2e": "NODE_OPTIONS=--max-old-space-size=16384 REACT_APP_TERRAWARE_TIER=test start-server-and-test start http://127.0.0.1:3000 playwright:run",
```

Comment:
I moved off `start-server-and-test` because it assumes starting the server doesn't exit, but the way we have the `start-server.sh` script set up, it exits successfully and runs the server in the background.

It wasn't really giving us anything other than the call to `wait-on`, so I think removing it is OK.